projects/plugins/crm/src/automation/commons/triggers/contacts/class-contact-before-deleted.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Jetpack CRM Automation Contact_Before_Deleted trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Exception;
+use Automattic\Jetpack\CRM\Automation\Automation_Workflow;
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Contact_Before_Deleted class.
+ */
+class Contact_Before_Deleted extends Base_Trigger {
+
+	/**
+	 * Contructs the Contact_Before_Deleted instance.
+	 */
+	public function __construct() {
+		$trigger_data = array(
+			'name'        => 'contact_before_delete',
+			'title'       => 'Contact Before Deleted',
+			'description' => 'Triggered just before a contact is deleted.',
```

Comment:
These looks like they could benefit from translation 😄 

```suggestion
			'title'       => __( 'Contact Before Deleted', 'zero-bs-crm' ),
			'description' => __( 'Triggered just before a contact is deleted.', 'zero-bs-crm' ),
```